tee-worker/omni-executor/rpc-server/src/verify_auth.rs

```diff
@@ -79,7 +80,11 @@ pub fn verify_web3_authentication(
 	verification_code_storage
 		.remove(&storage_key)
 		.map_err(|_| AuthenticationError::VerificationCodeNotFound)?;
-	let message = HeimaMessagePayload { message_code };
+	let message = HeimaMessagePayload {
+		client_id: client_id.to_string(),
+		omni_account: signer.to_omni_account().to_string(),
```

Comment:
Does `to_string()` have `0x` prefix? Could you check? (We want `0x prefix)

Reply:
I'll get it merged and fix it in the loginUser PR, I need some changes from this there